### ISSUE TITLE
[PRE REVIEW]: ggeffects - Tidy Data Frames of Marginal Effects from Regression Models

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#771]
**Submitting author:** @strengejacke (<a href="http://orcid.org/0000-0002-8895-3206">Daniel Lüdecke</a>)
**Repository:** <a href="https://github.com/strengejacke/ggeffects" target ="_blank">https://github.com/strengejacke/ggeffects</a>
**Version:** 0.3.4
**Editor:** @leeper
**Reviewers:** @cdcrabtree

**Author instructions**

Thanks for submitting your paper to JOSS @strengejacke. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@strengejacke if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks. @leeper it looks like you're currently assigned as the editor for this paper :tada:

For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00771/joss.00771/10.21105.joss.00771.pdf)

**Comment by arfon** [comment]
:wave: @leeper - the submitting author suggested you as the handling editor.

**Comment by leeper** [comment]
@cdcrabtree Would you be willing to review this for JOSS?

**Comment by leeper** [comment]
@mdogucu Would you be willing to review this for JOSS?

**Comment by cdcrabtree** [comment]
@leeper I would, but I am traveling and probably can't begin on this until the weekend. Does that work for JOSS?

**Comment by leeper** [comment]
@cdcrabtree Totally fine. We normally ask for a review within two weeks, but can be a bit flexible.

**Comment by cdcrabtree** [comment]
@leeper In that case, I'm happy to review. How do I accept the request?

**Comment by strengejacke** [comment]
@whedon generate pdf
(I resized figures, so page breaks hopefully look better)

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[--> Check article proof :page_facing_up: <--](https://github.com/openjournals/joss-papers/blob/joss.00771/joss.00771/10.21105.joss.00771.pdf)

**Comment by leeper** [comment]
@whedon add @cdcrabtree as reviewer

**Comment by whedon** [comment]
OK, @cdcrabtree is now a reviewer

**Comment by leeper** [comment]
@cdcrabtree That's great. Thanks! I've just added you. I'll open up the review tomorrow (it will be a new issue on this repo) and you'll get a notification from it with instructions on how to proceed with the review.

**Comment by cdcrabtree** [comment]
@leeper Great. Thanks!

**Comment by strengejacke** [comment]
If you're still looking for reviewers, @Ohlsen has agreed to review this paper.

**Comment by leeper** [comment]
@whedon start review magic-word=bananas

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/772. Feel free to close this issue now!

**Comment by leeper** [comment]
@strengejacke Thanks. I'll proceed with one reviewer at this point, but might rope in a further reviewer.
